### PR TITLE
First main merge

### PR DESCRIPTION
**Pull request by out-spaced**

### CODE REVIEW

**Review by spencerasava — APPROVED** [review]
ILL ALLOW IT

**Comment by ebbarker** [comment]
go for it. live and learn

**Review by zongchen-yang — APPROVED** [review]

File: client/components/overview/Overview.jsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include React from 'react';
```
Comment (ebbarker) [inline client/components/overview/Overview.jsx:1]: import?

Comment (out-spaced) [reply client/components/overview/Overview.jsx:1]: shutup

**Review by ebbarker — APPROVED** [review]

**Pull request merged**